app/templates/macros.html

```diff
@@ -41,8 +40,17 @@
         },
       };
 
-      var chart = new
-      google.visualization.LineChart(document.getElementById('{{ id }}'));
+      if('{{ type }}' == 'line'){
+        var chart = new google.visualization.LineChart(
+          document.getElementById('{{ id }}')
+        );
+      }
+      else{
```

Comment:
we might want to improve this logic in case of even more types of charts in the future